### PR TITLE
detect if a message was a haiku

### LINKED ISSUE #79: Duckbot should detect when a message is a haiku
Duckbot should count syllables for messages. If the message has the correct amount syllables for a haiku it should return that message formatted as a haiku.

> Dave: It's a little weird. I think "kids these days" are more needy in general.
> DuckBot: It's a little weird.
>                I think "kids these days" are more
>                Needy in general.

### PR DESCRIPTION
**Pull request by twentylemon**
Fixes #79 

~~I ended up finding [Carnegie Mellon Pronouncing Dictionary](http://www.speech.cs.cmu.edu/cgi-bin/cmudict), and it also happens to be on [github](https://github.com/Alexir/CMUdict/blob/master/cmudict-0.7b). It lists pronunciation and emphasis of some 130k words. They also have a [tool](http://www.speech.cs.cmu.edu/tools/lextool.html) for trying to pronounce new words not in the dictionary, which could be an extension to what I have done here.~~

nltk has the cmudict built in, so I'm using that now instead. nltk is a huge pain to stub out when testing, I couldn't figure out a way to get it running without downloading the packages (it literally checks when you import the package, so you can't even mock it out). I eventually just added the package download to the `install` script so it'd work.

### CODE REVIEW

**Comment by twentylemon** [comment]
Moving to draft. Seems nltk has the cmudict built in!
* https://github.com/nltk/nltk/blob/develop/nltk/corpus/reader/cmudict.py

I'll opt to use that instead of parsing it myself.

File: duckbot/cogs/messages/haiku.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+
+class Haiku(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+        self.syllables = {}
+
+    @commands.Cog.listener("on_ready")
+    async def build_syllable_dictionary(self):
+        nltk.download("cmudict")
```
Comment (twentylemon) [inline duckbot/cogs/messages/haiku.py:14]: Think a bit more about this download, maybe we should put this into the dockerfile for duckbot? What are your thoughts?  
We'd still need to download it for the `install` script, but it'd save this re-download every deployment.

Comment (twentylemon) [reply duckbot/cogs/messages/haiku.py:14]: Can apparently also use `setup.py` to install this kind of stuff. Maybe worth looking into later.

Comment (Chippers255) [reply duckbot/cogs/messages/haiku.py:14]: I'll dig up an old dockerfile we used to use at work, we were able to reload the dockerfile with the NLTK downloads.

Comment (twentylemon) [reply duckbot/cogs/messages/haiku.py:14]: Threw together something [on a dev branch](https://github.com/twentylemon/duckbot/pull/12/files) which uses `setuptools` to ensure the nltk data is downloaded. Think for now, we can just leave this PR as is, and revisit when I post a PR for that branch.

**Review by Chippers255 — APPROVED** [review]

**Pull request merged**